scram-tools.file/tools/rocm/rocm-rocrand.xml

```diff
@@ -0,0 +1,10 @@
+<tool name="rocm-rocrand" version="@TOOL_VERSION@">
+  <info url="https://github.com/ROCmSoftwarePlatform/rocRAND"/>
+  <use name="rocm"/>
+  <lib name="hiprand"/>
+  <lib name="rocrand"/>
+  <client>
+    <environment name="LIBDIR"  default="$ROCM_ROOT/lib"/>
```

Comment:
@fwyzard , you need to add
```
<environment name="ROCM_ROCRAND_BASE" default="@TOOL_ROOT@"/>
```
and use `$ROCM_ROCRAND_BASE` instead of `ROCM_BASE` in the toolfile

Reply:
Ah... so it cannot use `$ROCM_ROOT` from the `rocm` package ?

Reply:
no, scram treats each tool independently

Reply:
But, for example `$CUDA_ROOT` is available to the other spec files that depend on `cuda`:
https://github.com/cms-sw/cmsdist/blob/1e3431718a7f292c82df1428d3944649bc5e7479/ucx.spec#L36

Reply:
yes but this is in spec file which sources all the `etc/init.sh` files of its dependency. While processing toolfile can use @TOOL_ROOT@ every where in toolfile but then you end up with full @TOOL_ROOT@  path for all toolfile variable ( which means more editing is needed when you want update tool in your dev area to point to different location)